test/SlowTests/Issues/RavenDB-23167-Voron.cs

```diff
@@ -0,0 +1,200 @@
+﻿using System;
+using System.Linq;
+using Raven.Server.Documents;
+using Raven.Server.Documents.Revisions;
+using Sparrow.Server;
+using Sparrow.Server.Utils;
+using Tests.Infrastructure;
+using Voron;
+using Voron.Data.Tables;
+using Xunit;
+using Xunit.Abstractions;
+using Voron.Impl;
+using static Voron.Data.Tables.TableSchema;
+using Sparrow.Binary;
+using Sparrow.Threading;
+using FastTests.Voron;
+using Sparrow.Json;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_23167_Voron : StorageTest
```

Comment:
I want here two more tests with random data.
1. you insert 10 random elements
2. you insert 100 random elements

and you do seek backward from a random element as well.

check tests with `InlineDataWithRandomSeed`

Reply:
Done